src/client/index.scss

```diff
@@ -1,31 +1,39 @@
-h1 {
+.header1 {
```

Comment:
Why was this changed? Surely the site has a theming that means all h1 is this colour and therefore it would be a massive pain to have to import this every time I use an h1? I'm sure there's a good reason so let's discuss tomorrow :smile_cat: